plugins/generator-1.17.1/forge-1.17.1/procedures/world_data_canseesky.java.ftl

```diff
@@ -0,0 +1 @@
+(world.canSeeSky(new BlockPos((int)${input$x},(int)${input$y},(int)${input$z})))
```

Comment:
Based on forge-bot 1:1 mapping from 1.16.5 generator would be canSeeSkyFromBelowWater